app/src/renderer/components/common/TmSessionSignUp.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="tm-session">
-    <tm-form-struct :submit="() => onSubmit()" class="tm-session-container">
+    <tm-form-struct :submit="onSubmit.bind(this)" class="tm-session-container">
```

Comment:
separate PR
We can try a new method
```
:submit="wrapper(onSubmit)"

wrapper(func) {func(this)}
```

Reply:
https://github.com/cosmos/voyager/issues/1890